pkgs/development/libraries/qt-5/5.15/default.nix

```diff
@@ -174,6 +174,8 @@ let
       ./qtwebengine-darwin-no-platform-check.patch
       ./qtwebengine-mac-dont-set-dsymutil-path.patch
       ./qtwebengine-darwin-checks.patch
+    ] ++ lib.optionals stdenv.isLinux [
```

Comment:
Maybe this check could be relaxed to also account for BSDs and such? I'm not sure how many platforms we use Pulseaudio on...

Reply:
Actually it is possible to enable it unconditionally everywhere, as it's usage is guarded by the `use_pulseaudio` flag which itself is autodetected during build time on both versions.

```
  if (use_pulseaudio) {
    defines += [ "USE_PULSEAUDIO" ]
    if (!link_pulseaudio) {
      defines += [ "DLOPEN_PULSEAUDIO" ]
    }
  }
```